docker-compose.yml

```diff
@@ -0,0 +1,67 @@
+version: '3'
+
+services:
+    assets:
+        build:
+            context: .
+            dockerfile: Dockerfile.assets
+            args:
+                image_tag: ${IMAGE_TAG}
+        image: elifesciences/journal_assets:${IMAGE_TAG}
+        depends_on:
+            - composer
+            - npm
+    cli:
+        build:
+            context: .
+            dockerfile: Dockerfile.cli
+            args:
+                image_tag: ${IMAGE_TAG}
+                php_version: ${PHP_VERSION}
+        image: elifesciences/journal_cli:${IMAGE_TAG}
+        volumes:
+            - ./.docker/parameters.yml:/srv/journal/app/config/parameters.yml
+        depends_on:
+            - assets
+            - composer
+            - redis
+    composer:
+        build:
+            context: .
+            dockerfile: Dockerfile.composer
+            args:
+                composer_dev_arg: ${COMPOSER_DEV_ARG}
+        image: elifesciences/journal_composer:${IMAGE_TAG}
+        command: /bin/bash
+    fpm:
+        build:
+            context: .
+            dockerfile: Dockerfile.fpm
+            args:
+                image_tag: ${IMAGE_TAG}
+                php_version: ${PHP_VERSION}
+        image: elifesciences/journal_fpm:${IMAGE_TAG}
+        volumes:
+            - ./.docker/parameters.yml:/srv/journal/app/config/parameters.yml
+        depends_on:
+            - assets
+            - composer
+            - redis
+    npm:
+        build:
+            context: .
+            dockerfile: Dockerfile.npm
+        image: elifesciences/journal_npm:${IMAGE_TAG}
+    redis:
+        image: redis:3.2.10-alpine
```

Comment:
Matches the highest ElastiCache version. Different to what we currently use (`2.8.24`) as there's no official image for that version.